src/docs/environment.mdx

```diff
@@ -170,6 +172,106 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   displayed correctly.
 </Alert>
 
+## Setting up Getsentry
+
+<Alert title="Corporate Users Only" level="warning">
+  Only Sentry employees have access to getsentry.
+</Alert>
+
+Now that you have sentry all set up, it's time to set up getsentry. Getsentry is
+the application we deploy to production and contains additional functionality
+only offered in our SaaS offering.
+
+Let's start off by cloning the `getsentry` repository to be adjacent with your
+sentry repository:
+
+```bash
+# Go to where you have sentry and clone getsentry.
+cd /path/to/sentry
+cd ..
+git clone git@github.com:getsentry/getsentry.git
+```
+
+It's necessary to keep getsentry in an adjacent directory (it's expected by
+`install-develop-requirements` in the getsentry Makefile). For example, if you
+did a `ls ~/code` you'd see something like:
+
+```bash
+sentry/   getsentry/
+```
+
+Next, create a virtualenv for `getsentry`, and install its dependencies:
+
+```bash
+# Create and source a virtualenv
+
+python -m virtualenv .venv
+source .venv/bin/activate
+
+# Install dependencies
+# Note that you can safely ignore any warnings from pip
+# about "you'll have version ... which is incompatible"
+# Regarding google cloud dependencies.
+
+make develop
+
+# Initialize and bootstrap the database
+make reset-db
+
+# Run the development server
+getsentry devserver --workers
+```
+
+Once you've booted the server you'll access the development environment
+via [http://dev.getsentry.net:8000](http://dev.getsentry.net:8000/), which will
+allow you to use the appropriate subdomains (app and www).
+
+You can set your local instance's org to use a business plan by running the
+following in getsentry:
+
+```go
+./bin/mock-subscription <org_slug> --plan mm2_a_500k
+```
+
+# How does Getsentry development work?
+
+Getsentry and sentry are configured with different Django settings. For
+getsentry, the settings are configured in `getsentry/conf/settings/defaults.py`.
+Additional environment specific overrides are found in files matching the
+environment name. For example, local development is found in
+`getsentry/conf/settings/dev.py`.
+
+For sentry, the settings file is installed at `~/.sentry/sentry.conf.py`
+(outside both repos). If you're investigating, or want to explore, discrepancies
+between the two files, those two files are a good place to start looking.
+
+If you're in your getsentry virtualenv, do a `pip freeze | grep sentry\@` and
```

Comment:
Everything from this line down can be removed. I wrote it all, but it's just too verbose of an explanation of what an editable install is, and it's basic knowledge anyways.